.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+PRs and issues are always welcome, if you have any questions or need help, feel free to open a new discussion on this project.
+
+## Common Commands
+
+### Running Tests
+
+This will create a `.tests` directory that contains all the plugin dependencies of neotest-kotlin.
+
+```shell
+make test
+```
+
+### Running formatter
+
+This requires that [StyLua](https://github.com/JohnnyMorganz/StyLua) is installed.
+
+```shell
+make format
+```
+
+## Modifying/Adding Test Detection
+
+In the [treesitter module](../lua/neotest-kotlin/treesitter) there are queries for deteching
+
+- Java package
+- Java class name
+- Test formats
+
+For adding new tests
+
+1. Create a new test in [example_project](../tests/example_project/) for your format
+2. use `:InspectTree` to view the current Treesitter output
```

Comment:
I referenced `:InspectTree` and `:EditQuery` in another comment, they make it so easy.